### PR TITLE
Jakob dev

### PR DESCRIPTION
**Pull request by JacobSelt**
Started the Django Project and extended the API documentation.

### CODE REVIEW

**Comment by JacobSelt** [comment]
- Is the name for the Django Project (kreuzenonline) okay?
- I extended the API documentation, but the responses aren't consistent yet and there are missing some paths

File: api/api.yaml [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+          type: string
+        email:
+          type: string
+        lastLogin:
+          type: string
+        semester:
+          type: integer
+        course:
+          type: string
+          description: Zahnmedizin or Humanmedizin
```
Comment (scraptux) [inline api/api.yaml:222]: I'd change this to int or enum. e.g. Humanmed.: 0 (so it's standard), Zahmed.: 1, both: 2

**Review by scraptux — COMMENTED** [review]
Name is okay. There are some missing or misleading paths but they'll be fixed as soon as we get to testing.
IMO it's ready to merge

File: api/api.yaml (hunk 1 above)
Comment (JacobSelt) [reply api/api.yaml:222]: I've changed it locally and will push it with my next commit.

**Pull request merged**